internal/entity/models/aliyun_test.go

```
@@ -307,27 +308,48 @@ func TestAliyunChatStreamlyWithSenderRejectsStreamFalse(t *testing.T) {
-		requestPath <- r.URL.Path
-		requestBody <- body
-		w.Header().Set("Content-Type", "audio/mpeg")
-		_, _ = w.Write([]byte("fake-mp3-bytes"))
+		if requestPath != nil {
+			requestPath <- r.URL.Path
+		}
+		if requestBody != nil {
+			requestBody <- body
+		}
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprintf(w, `{"output":{"audio":{"url":%q},"finish_reason":"stop"},"request_id":"req-1"}`, server.URL+"/audio.wav")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)